### DISCUSSION TITLE
Looking for the right cable

### DISCUSSION THREAD

**Discussion by ledermann** [raomin/ESPAltherma#435] [Q&A] [answered]
I am trying to connect an `M5StickC Plus 2` to a `Daikin Altherma 3 H MT ECH2O` which has an X10A port.  As I'm stuck with choosing the right cable, can anyone give me a tip on what exactly I should buy?

**Comment by philipgo** [comment, marked as answer]
I simply use these cables:
https://www.amazon.de/gp/product/B00QV7O052

**Reply by ledermann** [reply]
Thanks a lot, will try this out!

**Comment by scapman** [comment]
Hi.
Did it work?
I have an M5StickC Plus 2 but it does not work. I am now trying with an esp8266.

**Reply by ledermann** [reply]
Yes, it works fine for me. Since 2 weeks, the M5StickC PLUS2 gets the values from my Daikin and pushes them to the MQTT broker.

**Comment by scapman** [comment]
Did you use the env:M5StickCPlus in platformio for flashing the firmware?
Does the LCD screen work?
Thanks

**Reply by ledermann** [reply]
Make sure you have the latest version of ESPAltherma. It seems to me that support for this stick has been added [recently](https://github.com/raomin/ESPAltherma/pull/403).

I'm using `env:m5stickcplus2` (!) within VSCode, Extension [Platform IO](https://marketplace.visualstudio.com/items?itemName=platformio.platformio-ide) for flashing. Because the stick does not know my WLAN after unpacking, I used a USB-C-Cable to connect it first directly to my Mac. I'm now using `upload_protocol = espota`.

Yes, the LCD also works. After unpacking, there was some demo stuff preinstalled. After flashing ESPAltherma, it now shows the plain text "ESPAltherma", if I remember correctly. I haven't seen it for two weeks because it's now inside the Daikin behind the plastic flap ;-)

**Comment by scapman** [comment]
Thanks! I will give a try in the next few days!

Il ven 10 mag 2024, 19:55 Georg Ledermann ***@***.***> ha
scritto:

**Reply by scapman** [reply]
Everything is working fine and smooth!
It was just a matter of using the last versione of ESPAltherma.

Really thanks again!!
